### ISSUE TITLE
Rails 6 support

### ISSUE THREAD

**Issue by jesperronn** [jmazzi/crypt_keeper#175]
Given I update to rails 6 in my Gemfile:
```diff
diff --git a/Gemfile b/Gemfile
 
-gem 'rails', '~>5.2'
+gem 'rails', '~>6.0.0.beta2'

```

Then I will not be able to update to Rails 6 due to a limit in crypt_keeper. Error message (when running `bundle update rails`:

```
    crypt_keeper (~> 2.0.0.rc2) was resolved to 2.0.1, which depends on
      activesupport (< 5.3, >= 4.2)

    rails (~> 6.0.0.beta2) was resolved to 6.0.0.beta2, which depends on
      activesupport (= 6.0.0.beta2)
```

What are the plans to update this plugin and maintaining the releases?

**Comment by jmazzi** [comment]
Once it ships, I'll bump the version restrictions. It seems to be working OK with rails 6 

https://travis-ci.org/jmazzi/crypt_keeper/jobs/520477844
https://travis-ci.org/jmazzi/crypt_keeper/jobs/520477849

**Comment by jesperronn** [comment]
Thanks for testing the requirements in https://github.com/jmazzi/crypt_keeper/pull/176

You probably want to bump the requirements before final Rails 6 is out, since this can help users finding other gems that require update. 

For people testing, you could as a minimum restore the branch in #176, and give instructions on how to use (`gem "crypt_keeper", git: "git://github.com/jmazzi/crypt_keeper.git", branch: "rails6"` assuming the branch name)

But I'd prefer if You would release a new version with bumped requirements, if you have the time and see appropriate :)

**Comment by mtchavez** [comment]
Yeah. Can we have a beta or release candidate version pushed to be able to run through upgrade testing? If not, the rails 6 compatible branch, as @jesperronn mentions, works with minimal effort. Thanks for the work you've done with the gem.

**Comment by jesperronn** [comment]
@jmazzi would you mind updating the rails 6 branch to have a successful test showing CryptKeeper still works on Rails 6.0.rc2

Although I understand your standpoint for not releasing until Rails 6 is final, I really, urge you to make a pre-release, which will ease testing for folks like me and @mtchavez

**Comment by jesperronn** [comment]
Now that Rails 6 IS released, would you mind to update the gem and make a release?

How can I help? Can I make a pull-request?

**Comment by taylorwilliams** [comment]
Bump, would love to see this merged

**Comment by jesperronn** [comment]
> Bump, would love to see this merged

You are more than welcome to chip in and fix Travis build in the pull-request: https://github.com/jmazzi/crypt_keeper/pull/178

I ran into bit of a dead end

**Comment by taylorwilliams** [comment]
> > Bump, would love to see this merged
> 
> You are more than welcome to chip in and fix Travis build in the pull-request: #178
> 
> I ran into bit of a dead end

I've given it a good go, looking at this pr https://github.com/jmazzi/crypt_keeper/pull/176 which built successfully on Rails 6 (https://travis-ci.org/jmazzi/crypt_keeper/jobs/520477849), it looks like we cannot do anything without @jmazzi's intervention. Travis CI setting perhaps?

**Comment by masterkain** [comment]
news?
